crates/stages/src/stages/bodies.rs

```diff
@@ -453,9 +445,8 @@ mod tests {
             .tx()
             .commit(|tx| {
                 let mut tx_cursor = tx.cursor_write::<tables::Transactions>()?;
-                let (_, transaction) = tx_cursor.last()?.expect("Could not read last transaction");
+                tx_cursor.last()?.expect("Could not read last transaction");
```

Comment:
this is an unrelated q, under what circumstances would this panic?

Reply:
i'd say if the test was written incorrectly